src/composables/useUtilities.ts

```diff
@@ -13,31 +14,21 @@ const swrvState = {
   SUCCESS_HAS_DATA: 'SUCCESS_HAS_DATA',
   ERROR: 'ERROR',
   STALE_IF_ERROR: 'STALE_IF_ERROR',
-}
+} as const satisfies Record<string, SwrvState>
 
 export default function useUtilities() {
   const useRequest = <Data = unknown, Error = { message: string }>(
     key: IKey,
     fn?: fetcherFn<AxiosResponse<Data>>,
     config?: IConfig,
   ) => {
-    const useSwrvFn =
-      typeof useSWRV === 'function'
-        ? useSWRV
-        : () => ({
-          // must return refs for consistent return types
-          data: ref({}),
-          error: ref(),
-          isValidating: ref(false),
-          mutate: () => ({}),
-        })
-
```

Comment:
`useSWRV` should always be a function. Are there any known scenarios where we might consider taking the other branch? @adamdehaven

Reply:
TBH I'm not sure why this code was in place

Reply:
The only thing that comes to mind is whether this was added specifically for some testing purposes. I suggest removing it to check if any test cases fail.